src/btree/bt_sync.c

```diff
@@ -248,9 +266,56 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
                     btree->rec_max_txn = mod->rec_max_txn;
                 if (mod != NULL && btree->rec_max_timestamp < mod->rec_max_timestamp)
                     btree->rec_max_timestamp = mod->rec_max_timestamp;
+
+                /*
+                 * Check whether the tree is lookaside btree and verify whether the page contents
+                 * are obsolete or not by checking the visibility of the page stop time pair.
+                 */
+
+                if (is_las) {
+                    obsolete = false;
+
+                    __wt_ref_is_obsolete(session, walk, &obsolete);
+
```

Comment:
Remove both of these blank lines. Also, the first thing you do in `wt_ref_is_obsolete` is initialize `obsoletep` so you don't need to do it here also. This one should be removed.

Reply:
Fixed.